modules/python/src2/gen2.py

```diff
@@ -598,14 +600,15 @@ def __init__(self, namespace, classname, name, decl, isconstructor, known_classe
             self.rettype = ""
         self.args = []
         self.array_counters = {}
+        self.mat_single_channel = (namespace == "cv.dnn" and classname == "dnn_Net")
```

Comment:
I propose to extract it from implementation and put as array like here: https://github.com/opencv/opencv/blob/2028998e00ab94fa8e2e9d3d888cc2d769d25027/modules/python/src2/typing_stubs_generation/predefined_types.py#L20

Reply:
use `misc/python` instead.

Reply:
Thank you for your reviews.

---

@opencv-alalek Could you be more specific? Do you mean I should create a file (.py) and save the array there then `gen2.py` load it from there?